File: plugin-sample-java/src/main/java/com/bytex/android/plugin/sample/SampleContext.java
package com.bytex.android.plugin.sample;

import com.android.build.gradle.AppExtension;
import com.ss.android.ugc.bytex.common.BaseContext;

import org.gradle.api.Project;

public class SampleContext extends BaseContext<SampleExtension> {

    public SampleContext(Project project, AppExtension android, SampleExtension extension) {
        super(project, android, extension);
    }
}

File: plugin-sample-kotlin/src/main/java/com/bytex/android/plugin/sample/SampleContext.kt
package com.bytex.android.plugin.sample

import com.android.build.gradle.AppExtension
import com.ss.android.ugc.bytex.common.BaseContext
import org.gradle.api.Project

class SampleContext(project: Project?, android: AppExtension?, extension: SampleExtension?) :
    BaseContext<SampleExtension>(project, android, extension) {
}
